public/app/features/auth-config/AuthConfigPage.tsx

```diff
@@ -79,50 +82,32 @@ export const AuthConfigPageUnconnected = ({ providerStatuses, isLoading, loadSet
     <Page navId="authentication" subTitle={subTitle}>
       <Page.Contents isLoading={isLoading}>
         <h3 className={styles.sectionHeader}>Configured authentication</h3>
-        {!!enabledProviders?.length && (
-          <div className={styles.cardsContainer}>
-            {enabledProviders.map((provider) => (
+        {authProviders.map((provider) =>
+          alreadyConfiguredProviders.includes(provider) ? (
```

Comment:
```suggestion
          checkProviderStatus(provider) ? (
```
Or if you apply my previous suggestion:
```suggestion
          isProviderConfigured(provider) ? (
```